game_patch/misc/player.cpp

```diff
@@ -288,6 +288,16 @@ CodeInjection sr_load_player_weapon_anims_injection{
     },
 };
 
+ConsoleCommand2 toggle_gibs_cmd{
```

Comment:
My thought process was that it made more sense to have the command itself in `player.cpp` because issuing the command is a player action whereas the functions all apply to entities - perhaps that's flawed, though. I can move it if you like, no problem.

Reply:
I don't define all commands in `player.cpp`, it would be a bad idea to define all commands there just because it's a player action. Wouldn't scale well